.github/workflows/ci.yml

```diff
@@ -16,20 +16,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: ["2.4", "2.5", "2.6", "2.7"]
-        rails: ["5", "6"]
+        ruby: ["2.5", "2.6", "2.7", "3.0"]
```

Comment:
Removed Ruby 2.4 since it’s not supported anymore (and it’s not available with GitHub actions either) but added Ruby 3 instead :)

Reply:
sounds good to me!